Pyrado/pyrado/utils/experiments.py

```diff
@@ -221,6 +221,22 @@ def load_experiment(
         # Policy
         policy = pyrado.load(f"{args.policy_name}.pt", ex_dir, obj=algo.policy, verbose=True)
 
+    elif algo.name == "pddr":
+        # Environment
+        env = pyrado.load("env.pkl", ex_dir)
+        # Policy
+        policy = pyrado.load(f"{args.policy_name}.pt", ex_dir, obj=algo.policy, verbose=True)
+        # Evaluation
+        extra["rollouts"] = algo.rollouts
+
+    elif algo.name == "pddr_teachers":
```

Comment:
I am not a fan on the additional class for PDDR teacher training. Why? Because you create an algorithm which is actually not really a disctinct algorithm. What you want is to run a policy optimitation subroutine on one exact domain parameter set (I mean an env that has been set to it). This is precisely what SPOTA does when computing the references. I would much prefer if you put this into a function like it is done there. I know that the loading logic would need to me moved to PDDR then.
Is there a strong argument for keeping it like this?